### ISSUE TITLE
No warning in case when 2 ISets containing 2 nodes each are merged sharing same path from root

### ISSUE THREAD

**Issue by PARAG00991** [gambitproject/jsgte#86]
Steps to reproduce the issue:
1. Create a tree as shown below:
![screenshot from 2016-04-20 18 30 00](https://cloud.githubusercontent.com/assets/8044561/14675367/57d0854c-0726-11e6-8ea8-fec0b873b9d1.png)
2. Now, merge the nodes in ISets as shown below:
![screenshot from 2016-04-20 18 30 20](https://cloud.githubusercontent.com/assets/8044561/14675395/6dd25b0e-0726-11e6-9b12-655991fa5838.png)
3. Now, if we try to merge the 2 ISets containing 2 nodes each shown, we know that the 2 ISets have same no. of moves and both belong to same player but both share the same path from root.
![screenshot from 2016-04-20 18 30 38](https://cloud.githubusercontent.com/assets/8044561/14675461/bd63a52e-0726-11e6-8a71-9a7c8aaf6888.png)
If we click on 1 ISet and then on other, it will do nothing but if we keep trying to click alternately on 2 ISets we get warning as `Couldn't merge the information sets.Please select two information sets with same number of moves` which is still not appropriate. It should give us a warning as `Couldn't merge the information sets.Please select two information sets that do not share a path from root`.

**Comment by stengel** [comment]
Parag, many thanks for noticing all these issues and your patches - should this not be treated as a general case of "not merging info sets when the new info set would contain two nodes that share a path from the root"? Is this resolved by your earlier pull request 84?  

--Bernhard 
- Parag Jain notifications@github.com [20-04-2016 06:08:45 -0700]:

**Comment by PARAG00991** [comment]
Yeah, it can be considered as a general case as you mentioned and this issue needs to be resolved separately. #84 resolves only the case when new info set is being merged with root of the tree.
